Gemfile

```diff
@@ -102,3 +101,5 @@ group :cucumber, :test do
   gem "govuk_test"
   gem "launchy"
 end
+
+gem "terser", "~> 1.1"
```

Comment:
It might be nice to move this `gem "terser"` line up so it sits alongside the other 'ungrouped' gems.

I don't think this is strictly enforced (there doesn't seem to be a linter rule for it) but it seems like the existing convention is for ungrouped gems to sit together, and be listed alphabetically.

So for this gem I think it'd sit on line 59, underneath `sprockets-rails` and above `transitions`.

Reply:
makes sense - I'll also remove the `~-> 1.1` specification for consistency.